andela/Ah-frontend-guardians
# #162685634: Users should be able to share articles on social media.

**What does this PR do?**
This PR enables users to share articles on their social media profiles, that is, Twitter and Facebook.

**Description of Task to be completed?**
A user should be able to share an article on Twitter and Facebook.

**How should this be manually tested?**

- Run the following commands in your terminal, one at a time:
```
$ git clone https://github.com/andela/Ah-frontend-guardians.git
$ git pull origin ft-social-sharing-162685634
$ git checkout ft-social-sharing-162685634
$ npm install
$ npm start
```
- The application should be live at this point. Register and log in a new user and click on any article card on the `Home` page.
- You should be able to view the whole article after which you click the `Share` button in the bottom right of the article.
- Click on the social media icon of your choice to share the article.
